frontend/public/static/locale/fr/translate.ftl

```diff
@@ -0,0 +1,39 @@
+# Editor
+
+entitydetails-editor-button-copy = Copier
+entitydetails-editor-button-clear = Effacer
+entitydetails-editor-button-send = Envoyer
+
+# Plural Selector
+
+plural-selector-form-zero = zero
+plural-selector-form-one = un
+plural-selector-form-two = deux
+plural-selector-form-few = un peu
+plural-selector-form-many = beaucoup
+plural-selector-form-other = autre
```

Comment:
I understand why you would want to keep these in English, but I also think that can be applied to most of the rest of the application (as we use domain-specific language in many places). I believe the whole point of this is to be able to translate strings like these to make them clearer to translators. 

What's your rationale for not translating them?

Reply:
(The following is my own interpretation. We might want localizers' feedback on this because ultimately, this UI is supposed to help them create better translations.)

These names are symbolic categories, not actual numerals. They just happen to be called after English numerals. For instance, translating `one` as `un` might be misleading because this category is also used with some decimals, e.g. _1,5 jour_. Other languages may have more complex rules. For instance in [Russian](http://www.unicode.org/cldr/charts/29/supplemental/language_plural_rules.html#ru), `one` is used for numbers which end with `1` but not with `11`. Translating the name of this category to `один` may suggest that it's only to be used with the number `1`.

Reply:
The culprit here is the usage of technical jargon in the UI. `two` isn't any more indicative than `Zwei` that this is meant to be a dual form. Or that `one` means _Singular_.

But .... replacing the short keys with actual grammatical Terms for the target language might lead to UI regressions, which we shouldn't introduce as part of this transition, I think.

My recommendation would be to not localize these terms now, and to file a follow-up to create a UI exposing the correct grammatical forms later on. And those human-targetting strings should then also be localizable.

Reply:
Note that we use two different UIs for plurals.

The one we talk about here is used for Gettext translations and works better with Machinery (and the rest of the UI). The other one is used by Fluent and allows (will allow) for features only supported in Fluent (e.g. custom variants, default variants, having sibling elements).

I agree it's best to not localize this for now.

Reply:
Thanks everyone for your inputs! I shall not make these localizable for now. :-)